### PR TITLE
hebrew letters

### PR DESCRIPTION
**Pull request by uriva**

### CODE REVIEW

File: doctr/datasets/vocabs.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
     "digits": string.digits,
     "ascii_letters": string.ascii_letters,
     "punctuation": string.punctuation,
-    "currency": "£€¥¢฿",
+    "currency": "£€¥¢฿₪",
```
Comment (felixT2K) [inline doctr/datasets/vocabs.py:16]: @uriva This needs to be added to Line 47 otherwise it would break our existing models :)

**Review by felixT2K — CHANGES_REQUESTED** [review]
Thansk for the PR @uriva only one minor change please :)

**Comment by uriva** [comment]
ptal

**Review by felixdittrich92 — APPROVED** [review]
Thanks :)

**Pull request merged**